news.js

```diff
@@ -0,0 +1,244 @@
+    }
+
+];
+//Channels present in the dropdown list
+var channels = ["BBC", "Aaj Tak", "NBC", "NDTV", "Republic", "TV9", "India Today", "Times Now", "Fox News", "CNN"];
+//variable used to store the complete data of all news channels
+var fulldata = " ";
+
+//Function to display the selected news category
+function myFunction() {
+    var x = document.getElementById("sel-category").value;
+    for (let i = 0; i < data.length; i++) {
+        if (x === data[i].heading) {
+            let display = "<div class='content' id='content'>" +
+                "<div class='content__sub' id='content__display'>" +
+                data[i].image +
+                "<h3 class='content__modifier content__head'>" + data[i].heading + "</h3>" +
+                "<p class='content__modifier content__date'>" + data[i].date + "</p>" +
+                "<p class='content__modifier content__matter'>" + data[i].content + "</p> " +
+                "<a href='#' class='content__modifier btn btn--pink' id='myBtn' onclick='showpopup()'>Continue Reading</a>" +
+                "</div>" +
+                `<div id="myModal" class="modal">
+                <div class="modal-content">
+                  <div class="modal-header">
+                    <span class="close" onclick="closepopup()">&times;</span>
+                    <h2 id="popup_head">`+ data[i].heading + `</h2>
+                  </div>
+                  <div class="modal-body">
+                    <p id="popup_content">`+ data[i].popup + `</p>
+                  </div>
+                  <div class="modal-footer">
+                    <h3 id="popup_foot">&copy; NewsFeed 2019</h3>
+                  </div>
+                </div>
+                </div>`+
+                "</div>";
+            document.getElementById("displaynews").innerHTML = display;
+        }
+        else if (x === "All") {
+            document.getElementById("displaynews").innerHTML = fulldata;
+        }
+    }
+}
+
+//On Load Function Call
+function displayPage() {
+    header();
+    footer();
+    displayForm();
+    displayNews();
+
+    function header() {
+        let html = '<h1 class="header__heading">NEWSFEED</h1><p class="header__caption">Yet another newsfeed</p>';
+        document.getElementById("header").innerHTML = html;
+    }
+
+    function footer() {
+        let html = ' <p class="footer__copyright">&copy; NewsFeed 2019</p>';
+        document.getElementById("footer").innerHTML = html;
+    }
+
+    function displayForm() {
+        var form = document.createElement("div");
+        form.setAttribute("id", "iamform");
+        let main = document.getElementById("main");
+        main.appendChild(form);
+
+        var allChannels = '';
+        for (let i = 0; i < channels.length; i++) {
+            allChannels += `<option value='` + channels[i] + `'>` + channels[i] + `</option>`;
+        }
+
+        document.getElementById("iamform").innerHTML =
+            "<div class='form'>" +
```

Comment:
use template literals instead of concat of strings. For instance if you wish to have 
Hi
There
you can do 
`Hi
There`
No need of "Hi" + "\nThere"

Reply:
Added Template literals